dashboard/test/lib/services/registration_reminder_test.rb

```diff
@@ -60,10 +59,6 @@ class Services::RegistrationReminderTest < ActiveSupport::TestCase
     end
   end
 
-  test 'applications_needing_first_reminder omits applications with no registration email' do
```

Comment:
I couldn't figure out what this test was doing, so I deleted it. Since registration reminders are sent to applicants who have already been accepted via email, they have a registration email